win/pkgs/vsbuildtools.ps1

```diff
@@ -0,0 +1,28 @@
+#Requires -RunAsAdministrator
+
+$ErrorActionPreference="Stop"
+
+$vs_where = "${Env:ProgramFiles(x86)}/Microsoft Visual Studio/Installer/vswhere.exe"
+if (Test-Path $vs_where)
+{
+    $vs_home = & $vs_where -latest -products * -property installationPath
+    if ($vs_home)
+    {
+        Write-Host "Found existing VS installtaion under `"$vs_home`". Skip installation."
+        exit 0
+    }
+}
+
+Invoke-WebRequest -Uri https://aka.ms/vs/16/release/vs_buildtools.exe -OutFile "${Env:SCRATCH}/vs_buildtools.exe"
+
+$install_cmd = "`"${Env:SCRATCH}/vs_buildtools.exe`" --quiet --wait --norestart --nocache " `
+    + "--add Microsoft.VisualStudio.Workload.VCTools;includeRecommended "                   `
+    + "--add Microsoft.VisualStudio.Component.Windows10SDK.16299 "                          `
```

Comment:
Also add latest WinSDK.

Reply:
It's installed by default with `includeRecommended` option.

You can check the workloads from:
https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2019&preserve-view=true